outputs.md

```diff
@@ -132,12 +140,20 @@ Text file of generated bandpass solutions. The first column is the frequency cha
       * **flag**: type codes where 0 is no flag, 1 is low confidence, and 2 is sidelobe contamination.
       * **extend**: a pointer. For extended sources, the pointer references a new source_array structure containing all of the extended components, in the same format. For point sources, it is a Null pointer.
       * **flux**: structure of the fluxes for the source. The order of potential polarizations is xx, yy, xy, and yx in apparent brightness or I, Q, U, V in sky brightness. Deconvolution only operates in Stokes I so Q=U=V=0.
-  
+
   * **source_mask**<br />
-  
+
   * **weights_arr**<br />
 
 ### \<obsids\>\_fhd_params.sav <br />
+  * **params**: Structure with arrays that contain various metadata in attributes listed below.
```

Comment:
I had quite a bit of confusion when trying to outline these below. I've marked them for help. If these are obvious to someone, maybe fill them in now. Otherwise leave as is.